common/persistence/cassandraPersistenceClientFactory.go

```diff
@@ -30,14 +30,14 @@ import (
 type (
 	cassandraPersistenceClientFactory struct {
 		session       *gocql.Session
-		metricsClient metrics.Client
 		logger        bark.Logger
+		metricsClient metrics.Client
```

Comment:
revert this

Reply:
we need this field, so the factory can use the metrics client if provided

Reply:
fixed